components/raftstore-v2/src/operation/command/write/mod.rs

```diff
@@ -228,4 +233,35 @@ impl<EK: KvEngine, R> Apply<EK, R> {
         // TODO: reuse the same delete as split/merge.
         Ok(())
     }
+
+    #[inline]
+    pub fn apply_ingest(&mut self, ssts: Vec<SstMeta>) -> Result<()> {
+        PEER_WRITE_CMD_COUNTER.ingest_sst.inc();
```

Comment:
shall we add apply_ingest duration metrics？maybe we should reuse INGEST_SST_DURATION_SECONDS

Reply:
It's already observed by `IMPORTER_INGEST_DURATION`.